ouroboros-network/test/Test/Mux.hs

```diff
@@ -1,17 +1,23 @@
+{-# LANGUAGE DataKinds           #-}
+{-# LANGUAGE TypeFamilies        #-}
+{-# LANGUAGE RankNTypes          #-}
 {-# LANGUAGE ScopedTypeVariables #-}
-{-# OPTIONS_GHC -Wno-orphans #-}
+
+{-# OPTIONS_GHC -Wno-orphans     #-}
 
 module Test.Mux (
-      TestProtocols1 (..)
+      NetworkMagic (..)
+    , NetworkMagicWithBufSize (..)
```

Comment:
Is this happening? I thought that the agreed upon use case was an option to only be a client.

Reply:
Could you expand what you mean?

Reply:
You clarified  bellow that this was a toy example.
Anyway in the meeting on Wednesday we agreed that there was no need for buffer size negotiation but that we should add a client-flag-only as an option for version negotiation. The flag would indicate that the client would only be able to act as initiator for all miniprotocols.